src/mewbot/io/rss.py

```diff
@@ -0,0 +1,514 @@
+#!/usr/bin/env python3
+
+from __future__ import annotations
+
+import asyncio
+from typing import Optional, Set, Sequence, Type, Any, Union, List, SupportsInt
+
```

Comment:
This is a personal preference thing, but I like to keep typing separate from the other imports

```suggestion
from typing import Optional, Set, Sequence, Type, Any, Union, List, SupportsInt

import asyncio
```

Reply:
Updated to seperate